testutil/src/main/java/org/itsallcode/openfasttrace/testutil/CompareAssertions.java

```diff
@@ -0,0 +1,129 @@
+package org.itsallcode.openfasttrace.testutil;
+
+/*-
+ * #%L
+ * OpenFastTrace Test utilities
+ * %%
+ * Copyright (C) 2016 - 2020 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Comparator;
+
+public class CompareAssertions
```

Comment:
I think this class deserves some JavaDoc. It appears out of nowhere, so a little hint on the purpose would be nice. :-)

Reply:
✅